bin/semver.js

```diff
@@ -178,6 +178,13 @@ Options:
 --ltr
         Coerce version strings left to right (default)
 
+-n <base>
+        Prerelease Identifier Base
+        that will let you let your prerelease number as 
+        zero-based or one-based. If you provide false
+        zero will be omitted. If you do not specify 
+        this parameter, it will default to zero-based.
+
```

Comment:
```suggestion
        that will let you let your prerelease number as 
        zero-based or one-based. If you provide false
        zero will be omitted. If you do not specify 
        this parameter, it will default to zero-based.

```